Fix typos and grammar errors across documentation (#144709)
python/cpython: Doc/library/trace.rst

   Display the version of the module and exit.

.. versionadded:: 3.8

>     Added ``--module`` option that allows you to run an executable module.

Reviewer: ```suggestion
    Added ``--module`` option that allows running an executable module.
```
